src/components/Table/TaskLink.tsx

```diff
@@ -1,17 +1,22 @@
+import { Body } from "@leafygreen-ui/typography";
 import { StyledRouterLink, WordBreak } from "components/styles";
 import { getTaskRoute } from "constants/routes";
+import { executionAsDisplay } from "pages/task/util/execution";
 
 interface TaskLinkProps {
+  execution: number;
+  onClick?: (taskId: string) => void;
   taskId: string;
   taskName: string;
-  onClick?: (taskId: string) => void;
 }
 export const TaskLink: React.VFC<TaskLinkProps> = ({
+  execution,
+  onClick = () => {},
   taskId,
   taskName,
-  onClick = () => {},
 }) => (
   <StyledRouterLink onClick={() => onClick(taskId)} to={getTaskRoute(taskId)}>
     <WordBreak>{taskName}</WordBreak>
+    <Body>Execution {executionAsDisplay(execution)}</Body>
```

Comment:
We only need to show this label on the execution tasks table (not a patch's tasks table) since the project focuses on restarting failed execution tasks thereby making those runs out of sync. I think the designs show the label on the wrong table so I just checked with Annie and confirmed this is the case.

I also think we could maybe only show the label if there is more than one execution for any execution task, but maybe that's something we should confirm in design review?